lib/inputstreamhelper.py

```diff
@@ -58,6 +74,11 @@ def __init__(self, protocol, drm=None):
         self.protocol = protocol
         self.drm = drm
 
+        try:
+            self._log('Platform information: {0}'.format(platform.uname()))
+        except IOError:  # Survive [Errno 10] No child processes
+            self._log('Platform information: undetermined')
```

Comment:
I know it was here before but is it useful to log this info in the contructor? Maybe this can be moved to the function where we need this.

Reply:
I am a proponent to review the logging in general. This is probably best looked at when we do unit and integration tests (with full logging enabled).